app/Http/Middleware/TrustProxies.php

```diff
@@ -19,5 +19,10 @@ class TrustProxies extends Middleware
      *
      * @var int
      */
-    protected $headers = Request::HEADER_X_FORWARDED_ALL;
+    protected $headers =
```

Comment:
what is the rationale of this change ?

Reply:
Please refer https://laravel.com/docs/9.x/upgrade section Trusted Proxies